### PR TITLE
Add PubSubEmulatorContainer

### PR DESCRIPTION
**Pull request by necosta**
Added a new test container for Google Pub/Sub emulator.

As this is my first contribution to this repo, let me know if there are any guidelines I need to follow.

Ready for review.

### CODE REVIEW

File: modules/gcloud/src/test/scala/com/dimafeng/testcontainers/PubSubEmulatorContainerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+import java.util.concurrent.TimeUnit
+import scala.collection.JavaConverters._
+import scala.concurrent.TimeoutException
+
+class PubSubEmulatorContainerSpec extends AnyWordSpecLike with Matchers with ForAllTestContainer {
+
+  override val container: PubSubEmulatorContainer = PubSubEmulatorContainer()
```
Comment (dimafeng) [inline modules/gcloud/src/test/scala/com/dimafeng/testcontainers/PubSubEmulatorContainerSpec.scala:15]: I see that `defaultImageName` uses the latest tag which is fine for the general use. But can we pin the version in the test (so it's not gets broken unpredictably)?

File: modules/gcloud/src/test/scala/com/dimafeng/testcontainers/PubSubEmulatorContainerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+  override def beforeStop(): Unit = {
+    container.topicAdminClient.close()
```
Comment (dimafeng) [inline modules/gcloud/src/test/scala/com/dimafeng/testcontainers/PubSubEmulatorContainerSpec.scala:18]: I'm curious if `container.topicAdminClient.close()` and `container.subscriptionAdminClient.close()` can be done in the `container.close()`

**Comment by dimafeng** [comment]
@necosta thank you! I left a couple of quick comments.

File: modules/gcloud/src/test/scala/com/dimafeng/testcontainers/PubSubEmulatorContainerSpec.scala (hunk 1 above)
Comment (necosta) [reply modules/gcloud/src/test/scala/com/dimafeng/testcontainers/PubSubEmulatorContainerSpec.scala]: Sounds reasonable yes. I locked version `382.0.0` in latest commit so image is unaltered.

File: modules/gcloud/src/test/scala/com/dimafeng/testcontainers/PubSubEmulatorContainerSpec.scala (hunk 2 above)
Comment (necosta) [reply modules/gcloud/src/test/scala/com/dimafeng/testcontainers/PubSubEmulatorContainerSpec.scala]: Good point. I changed the logic so it's done in `PubSubEmulatorContainer` before closing the container.

**Pull request merged**